lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -681,6 +681,12 @@ sub overview {
     );
     my @latest_jobs = $self->schema->resultset('Jobs')->complex_query(%$search_args)->latest_jobs($until);
     ($stash{archs}, $stash{results}, $stash{aggregated}) = $self->prepare_job_results(\@latest_jobs);
+ 
+    # Allow n queries per minute, per user (if logged in)
+    my $lockname = 'webui_overview_rate_limit';
+    if (my $user = $self->current_user) { $lockname .= $user }
```

Comment:
I would expect that this only happens the address of the user object.

Reply:
That's a string, like `cdywan`, if logged in

Reply:
I don't think so, it'll likely just be something like `OpenQA::Schema::Result::Users=HASH(0x563a8d87dd40)`.

Reply:
This is already used here:
https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/WebAPI/Controller/API/V1/Search.pm#L186

So either this is also wrong, or the user is a string :)

Reply:
Yeah, I thought it must be sound since I took it from existing code... I guess we never had tests to back it up. I tried to add new unit tests but failed to figure out how to get authenticated API calls working that have a username in the controller.
And I have a fix for the existing code in any case: #3996